home/tests/test_content_import_export.py

```diff
@@ -1303,6 +1316,29 @@ class TestExport:
         container for related tests.
     """
 
+    def test_ordered_content_set_export(self, impexp: ImportExport) -> None:
+        """
+        Ordered Content Sets should export all pages correctly
+        """
+        ordered_content_set = OrderedContentSet(name="Test Title")
+        ordered_content_set.save()
+
+        content = impexp.export_ordered_content()
+        # Export should succeed
+        assert content is not None
+
+    def test_ordered_content_XLSX_export(self, xlsx_impexp: ImportExport) -> None:
+        """
+        Ordered Content Sets should export in XLSX format correctly
+        """
+        ordered_content_set = OrderedContentSet(name="Test Title")
+        ordered_content_set.save()
+
+        # exports in xlsx format
+        content = xlsx_impexp.export_ordered_content()
+
+        assert content is not None
```

Comment:
I think we need more comprehensive testing than this. We're not testing that the export is actually outputting sensible data that reflects what is in the database. And we're also not testing that if we export ordered content sets, that they'll be the same when we import them again.